src/GraphQL.Tests/Execution/AbstractTypeTests.cs

```diff
@@ -13,6 +13,16 @@ public void throws_when_unable_to_determine_object_type()
         error.InnerException.ShouldNotBeNull();
         error.InnerException.Message.ShouldBe("Abstract type Pet must resolve to an Object type at runtime for field Query.pets with value '{ name = Eli }', received 'null'.");
     }
+
+    [Fact]
+    public void throws_when_unable_to_determine_object_type_nullable()
```

Comment:
This test would have failed previously, since there was a bug in ExecutionStrategy that prevented checking values for nullable interface fields.

Reply:
That is, on the contrary - for non-nullable interface fields.